platform/darwin/test/MGLOfflineStorageTests.mm

```diff
@@ -21,8 +21,8 @@ + (void)tearDown {
                                                              appropriateForURL:nil
                                                                         create:NO
                                                                          error:nil];
-    // Unit tests don't use the main bundle; use com.mapbox.ios.sdk instead.
-    NSString *bundleIdentifier = [NSBundle bundleForClass:[MGLMapView class]].bundleIdentifier;
+    // As of iOS SDK 12.2 unit tests now have a bundle id: com.apple.dt.xctest.tool
```

Comment:
I think we can safely omit these comments in this file, since we’re now pointing to a shared external method (that itself better documents the reasoning).

Reply:
Agreed.